packages/client/src/profile/Profile.ts

```diff
@@ -916,4 +918,34 @@ export class Profile {
       await this.sdk.RemoveProfileInterest({ request }, headers);
     });
   }
+
+  /**
+   * Fetch profile guardian settings for the specified profile.
+   *
+   * ⚠️ Requires authenticated LensClient with the provided profileId.
+   *
+   * @param request - Request object for the mutation
+   * @returns {@link PromiseResult} with {@link ProfileGuardianResultFragment}
+   *
+   * @example
+   * ```ts
+   * await client.profile.profileGuardian({
```

Comment:
maybe let's call it just `guardian` to avoid repetition of "profile", so the call will look like

```
await client.profile.guardian({})
```

Reply:
I like it